cmd/webhook-manager/main.go

```diff
@@ -58,6 +58,10 @@ func main() {
 		klog.Fatalf("Configured port is invalid: %v", err)
 	}
 
+	if err := config.ReadCAFiles(); err != nil {
+		klog.Fatalf("Failed to read CA files: %v", err)
+	}
+
 	if err := app.Run(config); err != nil {
```

Comment:
I think It's better to call `config.ReadCAFiles()` in `cmd/webhook-manager/app/server.go`